src/inputs/Table/components/TableToolbar/TableToolbar.js

```diff
@@ -26,4 +34,44 @@ TableToolbar.propTypes = {
    * Function used to translate parent table element in fullscreen
    */
   toggleFullscreen: PropTypes.func.isRequired,
+  /*
+   * Boolean value to know if there is some data fully loaded
+   */
+  isReady: PropTypes.bool.isRequired,
+  /*
+   * Boolean use to know if the user have rights to edit the solution
+   */
+  editMode: PropTypes.bool.isRequired,
+  /*
+   * Function used to import a file in Table. If it's not defined, he do not display the import button
```

Comment:
need to rephrase _he do not display_
The same for line 50 where _export button_ is replaced by _import button_